evergreen.py

```diff
@@ -77,20 +78,29 @@ def main():  # pragma: no cover
         if repo.visibility.lower() not in filter_visibility:
             print("Skipping " + repo.full_name + " (visibility-filtered)")
             continue
+        existing_config = None
         try:
-            if repo.file_contents(".github/dependabot.yml").size > 0:
-                print(
-                    "Skipping " + repo.full_name + " (dependabot file already exists)"
-                )
-                continue
+            existing_config = repo.file_contents(".github/dependabot.yml")
```

Comment:
Extract to function with file name as argument?

since exactly like changes below except file name?

Reply:
[386e781](https://github.com/github/evergreen/pull/140/commits/386e78180aaf087e191e745063474d6e25748053)